.gitlab/workflows.yml

```diff
@@ -46,28 +41,98 @@ Python Wheels:
   tags:
     - $runner
   interruptible: false
+  artifacts:
+    paths:
+      - artifacts
+    expire_in: 1 day
   script:
-    - echo "Nothing implemented yet."
+    - export PROJECT_SRCS_DIR=$PWD/.. 
+    # Install GitHub CLI
```

Comment:
nit: it might be nice to move the download of the wheels into a separate job for better readability.

Reply:
I've separated this step into the `before_script` section to improve readability a bit while keeping it simple :-)